src/heapdump/heapdump.cc

```diff
@@ -13,7 +13,7 @@
 // OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 #include "node.h"  // Picks up BUILDING_NODE_EXTENSION on Windows, see #30.
-
+#include "nan.h"
```

Comment:
consider grabbing the most recent https://github.com/bnoordhuis/node-heapdump/blob/master/src/heapdump.cc

Reply:
+1 Richard and I had discussed this, we just need to check we are able to take a new drop.

Reply:
opened a new issue https://github.com/RuntimeTools/appmetrics/issues/579